src/index.html

```diff
@@ -10,8 +12,155 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
```

Comment:
[checklist ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item, format your tags correctly, fix it everywhere
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
     >
```